client/app.jsx

```diff
@@ -88,6 +89,9 @@ export default class App extends React.Component {
     if (route.path === 'search') {
       return <SearchPage searchResults={this.state.searchResults} />;
     }
+    if (route.path === 'airing') {
+      return <Airing airing={this.state.airing} />;
+    }
```

Comment:
I recommend refactoring your routes so that each page fetches its own data.

Load the airing anime in componentDidMount for Airing. And modify your search form to do this
```js
handleSubmit(event) {
  event.preventDefault();
  window.location.hash = 'search?keyword=' + this.state.search;
}
```

Pass the keyword to the SearchPage as a prop, and then fetch the anime in SearchPage componentDidMount

please lemme know if I can clarify. The goal is to have each page load its own data where possible instead of loading up everything in App